src/nylo/token.py

```diff
@@ -1,18 +1,44 @@
+# This file is a part of nylo
```

Comment:
token.py should not have the copyright. Only file beggining with __ shall have them.